daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md

```diff
@@ -77,6 +81,8 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | autoDeleteOnIdleInSec | N  |Time in seconds to wait before auto deleting messages. | `10`
 | maxReconnectionAttempts | N  |Defines the maximum number of reconnect attempts. Default: `30` | `30`
 | connectionRecoveryInSec | N  |Time in seconds to wait between connection recovery attempts. Defaults: `2` | `2`
+| publishMaxRetries | N  | The max number of retries for when Azure Service Bus responds with "too busy" in order to throttle messages. Defaults: `5` | `5`
```

Comment:
The description implies this only happens for throttling, is that the case? my understanding is that at least right now we don't really know the cause of an error that requires a retry (so this really applies to any retry flow even if not started due to throttling)

Reply:
We only retry when we get an error from the ASB SDK indicating the server is "too busy". We made the decision to be specific in this case.